bin/test.php

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use Translations\Tests\AbstractTranslationTestCase;
+
+class TestCase extends AbstractTranslationTestCase
+{
+    public function getTestCaseClassname(string $locale): string
+    {
+        foreach (static::$locales as $classname => $localCode) {
+            if ($localCode === $locale) {
+                $namespaces = explode('\\', $classname);
+                return end($namespaces);
+            }
+        }
+
+        throw new RuntimeException('Locale not found: ' . $locale);
```

Comment:
I think it would be better to be `Locale "<locale>" not found.`

Reply:
Fixed.